src/rdf4cpp/rdf/datatypes/registry/util/CharConvExt.hpp

```diff
@@ -124,41 +131,46 @@ std::string to_chars(F const value) noexcept {
     static constexpr size_t buf_sz = 5 + std::numeric_limits<F>::max_digits10 + std::max(2ul, detail::log10ceil(std::numeric_limits<F>::max_exponent10));
     std::array<char, buf_sz> buf;
 
-    std::to_chars_result res = [&]() {
-        [[maybe_unused]] F integral;
-        F const fraction = std::modf(value, &integral);
-
-        if (fraction == 0) {
-            // include one zero after dot
-            return std::to_chars(buf.data(), buf.data() + buf.size(), value, std::chars_format::scientific, 1);
-        } else {
-            // automatic precision guarantees round-trip-safety (https://en.cppreference.com/w/cpp/utility/to_chars)
-            return std::to_chars(buf.data(), buf.data() + buf.size(), value, std::chars_format::scientific);
-        }
-    }();
+    std::to_chars_result res = std::to_chars(buf.data(), buf.data() + buf.size(), value, std::chars_format::scientific);
```

Comment:
> [Unlike other formatting functions in C++ and C libraries, std::to_chars is locale-independent, non-allocating, and non-throwing.](https://en.cppreference.com/w/cpp/utility/to_chars#Notes)

Does locale=de-de_utf8 cause `,` being used as decimal separator?

Reply:
No it does not. See https://godbolt.org/z/dr5nYEKen